python/seldon_core/utils.py

```diff
@@ -27,19 +35,14 @@
 logger = logging.getLogger(__name__)
 
 
-ENV_MODEL_NAME = "PREDICTIVE_UNIT_ID"
-ENV_MODEL_IMAGE = "PREDICTIVE_UNIT_IMAGE"
-NONIMPLEMENTED_MSG = "NOT_IMPLEMENTED"
-
-model_name = os.environ.get(ENV_MODEL_NAME, f"{NONIMPLEMENTED_MSG}")
-image_name = os.environ.get(
-    ENV_MODEL_IMAGE, f"{NONIMPLEMENTED_MSG}:{NONIMPLEMENTED_MSG}"
-)
-
-
 def get_request_path():
+    model_name = get_model_name()
```

Comment:
Conscious that it falls slightly outside the scope of this PR, but it would make sense to return `None` rather than `NONIMPLEMENTED_MSG`?

Reply:
I think some of these are sometimes used for a key in dictionary. `None` cannot be a key as far as I remember.

Reply:
it can but if we read it back it will be "None" and not `None` so we need to convert anyway